src/tools/prism/language-definition.ts

```diff
@@ -1,26 +1,29 @@
 import { OpenFgaDslThemeTokenType } from "../../theme/theme.typings";
 
 export const languageDefinition = {
-  [OpenFgaDslThemeTokenType.MODULE]: {
+  module: {
```

Comment:
Thanks @Siddhant-K-code!

Can you elaborate on the change from using the enums to strings? Not sure how that helps

Reply:
> Why this approach?
> In this, each token in the languageDefinition object has been given a more descriptive name that aligns with the default PrismJS tokens. The alias property is used to maintain the mapping to the original OpenFgaDslThemeTokenType enum values.
>
> This approach allows for easier customization of the highlighting colors by utilizing the default PrismJS tokens, while still keeping the internal mapping to the specific OpenFgaDslThemeTokenType enum values.



I've tried to explain this approach in PR description. Let me know, if anything is clear or I could explain in better 🙏🏼